frontend/src/components/Difficulty/DifficultyCard.tsx

```diff
@@ -21,6 +22,7 @@ interface Props {
   courseId: string;
   started?: boolean;
   phrase: string;
+  difficulty: 1 | 2;
```

Comment:
You can use an enum in here, it will bring clarity to what this difficulty mean.

For instance is 1 the more difficult? or is it 2?

If you use an enum that becomes clear

enum Difficulty {
  Easy = 1,
  Medium,
  Advanced
}

https://www.typescriptlang.org/docs/handbook/enums.html